back/src/domains/convention/ports/ConventionRepository.ts

```diff
@@ -16,4 +16,5 @@ export interface ConventionRepository {
     endedSince: Date,
     now?: DateString,
   ) => Promise<ConventionId[]>;
+  deleteOldConventions: (updatedBefore: Date) => Promise<ConventionId[]>;
```

Comment:
un objet permet de documenté la date de façon plus explicite à l'usage: 

```suggestion
  deleteOldConventions: (params: {updatedBefore: Date}) => Promise<ConventionId[]>;
```